main.go

```diff
@@ -0,0 +1,300 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	ee "github.com/primev/preconf_blob_bidder/internal/eth"
+	bb "github.com/primev/preconf_blob_bidder/internal/mevcommit"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	// Load the .env file before setting up the app
+	envFile := os.Getenv("ENV_FILE")
+	if envFile == "" {
+		envFile = ".env"
+	}
+	if _, err := os.Stat(envFile); err == nil {
+		if err := loadEnvFile(envFile); err != nil {
+			fmt.Fprintf(os.Stderr, "Error loading .env file: %v\n", err)
+			os.Exit(1)
+		}
+	}
+
+	zerolog.SetGlobalLevel(zerolog.InfoLevel)
+	log.Logger = log.Output(os.Stderr).With().Timestamp().Logger()
+
+	app := &cli.App{
+		Name:  "Preconf Bidder",
+		Usage: "A tool for bidding in mev-commit preconfirmation auctions for blobs and transactions",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
```

Comment:
If you export all flags to the `var` block, you can refer to them by their name instead of the string in the name.